### ISSUE TITLE
[NV-179] - Add new MailerSend provider

### ISSUE THREAD

**Issue by Agathe-Brusset** [notifirehq/notifire#84]
### **What?**

New MailerSend Email Provider

### **How?**

API documentation: [www.developers.mailersend.com](http://www.developers.mailersend.com/)\
Using the SDK: MailerSend Node.js.

### **Usage**

Example for MailerSend provider registration needs to be finalized.

```
await notifire.registerProvider(
  new MailerSendProvider({
    apiKey: process.env.SPARKPOST_API_KEY
  })
);
```

> I'm Agathe Brusset, project manager of MailerSend, a solution to send transactional emails via API or SMTP. We would love to connect MailerSend to Notifire. Would you consider integrating our solution into your app as an email provider?

> Please find more details on our website: [www.mailersend.com](http://www.mailersend.com/)\
> Our API documentation: [www.developers.mailersend.com](http://www.developers.mailersend.com/)

> Let us know when you create yourselves an account so we can approve you.

**Comment by scopsy** [comment]
I've update the issue to comply with our issue template for new providers

**Comment by sjschweber** [comment]
Hey can I pick up this issue @scopsy ?

**Comment by pearceman** [comment]
Hey @sjschweber, How r u? Any updates on the issue? Can we help with something?

**Comment by fredsossa1** [comment]
Hi, I can work on this

**Comment by scopsy** [comment]
Thanks @fredsossa1! Assigned you❤️🙏

**Comment by djabarovgeorge** [comment]
Closed by #389  
With @fredsossa1 help
